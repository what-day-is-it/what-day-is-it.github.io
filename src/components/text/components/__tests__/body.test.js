import React from 'react';
import { shallow } from 'enzyme';
import Body from '../body';

describe('Body', () => {
  const props = {
    size: 'large',
    children: <div>Body Text</div>,
    className: 'foobar',
    color: 'black'
  };
  const wrapper = shallow(<Body {...props} />);
  const noPropsWrapper = shallow(<Body />);

  it('should render', () => {
    expect(wrapper).toMatchSnapshot();
    expect(noPropsWrapper).toMatchSnapshot();
  });

  it('should render the props', () => {
    expect(wrapper.find('.foobar').exists()).toEqual(true);
    expect(wrapper.find('.body-large.black').exists()).toEqual(true);
  });
});
