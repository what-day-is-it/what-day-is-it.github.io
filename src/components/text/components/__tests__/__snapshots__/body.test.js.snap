// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body should render 1`] = `
<div
  className="
      body-large black
      weight-normal align-left
      foobar
    "
>
  <div>
    Body Text
  </div>
</div>
`;

exports[`Body should render 2`] = `
<div
  className="
      body-regular black
      weight-normal align-left
      
    "
/>
`;
